### PR TITLE
Add first version

### PR DESCRIPTION
**Pull request by damiencarol**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ADD nginx.conf /etc/nginx/
+
+COPY --from=builder  /app/build/ /usr/share/nginx/html/
+COPY --from=builder  /app/build/ /var/www/html/
+
+# Append "daemon off;" to the beginning of the configuration
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
+
+# Expose ports
+EXPOSE 90
```
Comment (madchap) [inline Dockerfile]: 90? or 9000?

File: nginx.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+events { worker_connections 1024; }
+
+http {
+    include    mime.types;
+    sendfile on;
+    server {
+        root /usr/share/nginx/html/;
+        index index.html;
+        server_name localhost;
+        listen 90;
```
Comment (madchap) [inline nginx.conf]: 90 seems like a very odd number to listen on and requires root.

**Review by madchap — CHANGES_REQUESTED** [review]
Should not use root. Let's create a user and bind to a higher port.

File: Dockerfile (hunk 1 above)
Comment (damiencarol) [reply Dockerfile]: it's internal (you map it when you instantiate the container), I will change it to be aligned with other containers of the project. Thanks for pointing it.

Comment (damiencarol) [reply Dockerfile]: FYI, the Django container use port 9000

**Review by madchap — APPROVED** [review]

**Pull request merged**